spec/models/user_spec.rb

```diff
@@ -1,5 +1,89 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  before :all do
+    @user = User.create(full_name: 'Example Example', email: 'example@example.com', bio: 'example bo', image: 'image')
+    @user2 = User.create(full_name: 'ExampleNew Example', email: 'example2@example.com', bio: 'example bo', image: 'image')
+  end
+
+  describe '#email' do
```

Comment:
When you say this: describe '#email' do, it means you are testing method caled 'email' from User class, 
and here you are trying to test every behaviour about email + something about full_name

Lets not mix things, see some best practices from here:https://www.betterspecs.org/
and also how model classes spec structure should be: https://gist.github.com/kyletcarlson/6234923